docs/notes.md

```diff
@@ -224,3 +224,90 @@ We agreed to use hyphen-sepraten name covention for custom Events i.e.
 ```
 
 
+### Backend Communication
+We decided to briefly explain flow and the general usage of Command and Query Managers which are the foundation of our communication with the backend.
+The decision was motivated to ease the onboarding process for future colleagues who happen to work on currently implemented or create new ones and also for us to help when we have to come back to them in future
+ 
+Communication is based on Command/Query Managers and Resolvers which were implemented to unify how data is fetched from the backend and also to group all these types of calls in one place to be easily accessed.
+That also put some abstraction on it which can be difficult to decode at first glance.
+ 
```

Comment:
I would also add a short explanation about the difference between a QueryManager and a CommandManager